build.gradle

```diff
@@ -433,12 +433,18 @@ gradle.projectsEvaluated {
 
     project.tasks.withType(Test) { task ->
       if (task != null) {
-        if (BuildParams.runtimeJavaVersion > JavaVersion.VERSION_17) {
+        if (BuildParams.runtimeJavaVersion > JavaVersion.VERSION_17 && BuildParams.runtimeJavaVersion <= JavaVersion.VERSION_23) {
           task.jvmArgs += ["-Djava.security.manager=allow"]
```

Comment:
Is `-Djava.security.manager=allow` needed anymore now that the `System.setSecurityManager` calls have been removed?

Reply:
No, not anymore :-)

Reply:
@andrross: @cwperks started taking a look at clean-ups of the ```-Djava.security.manager``` across all code bases in core. We think its not a lot of changes and we can do the clean-up (in a day or two) once the final PR from this POC is merged.

Reply:
> @andrross: @cwperks started taking a look at clean-ups of the -Djava.security.manager across all code bases in core. We think its not a lot of changes and we can do the clean-up (in a day or two) once the final PR from this POC is merged.

Thanks @kumargu, I should have the clean pull request to core tomorrow,  doing some final tests with JDK-24